### DISCUSSION TITLE
Docker FAQ 问题集锦

### DISCUSSION THREAD

**Discussion by Yidadaa** [ChatGPTNextWeb/NextChat#2104] [General]
这里用来存放常见的与 docker 部署相关的 issue 与讨论。

This is a place to store common issues and discussions related to Docker deployment.

**Comment by 8pizhou** [comment]
使用时经常出错，用不了

**Comment by feather-root** [comment]
启动不了，也看不出什么问题，不知道怎么排查

**Reply by berryyan** [reply]
一样额情况

**Comment by Screek1337** [comment]
Hey guys!
How to run How to run ChatGPT-Next-Web with docker compose?
I've got docker-compose.yaml from this repo, set up .env file with all variables, but keep getting "no service selected" error after running docker compose up

**Reply by iijiang** [reply]
`docker-compose --profile no-proxy up`
